util/exec.go

```diff
@@ -19,6 +19,9 @@ type Command struct {
 
 	// Directory defines the directory the execution should run in, without changing the working directory of the caller.
 	Directory string
+	// Env defines all environment variables for the executed command.
+	// If HOME and PATH are not included, they will be set automatically.
+	Env map[string]string
```

Comment:
Would say that the command inherits all env variables unless overwritten in the map.

Reply:
Wups. copy&pasta error